### PR TITLE
Bugfix/hero landing

### PR DESCRIPTION
**Pull request by Sonny97**
fixed the in hero landing the scroll down

image
image

and fixed the size

image

### CODE REVIEW

File: css/home.css [hunk 1]
```diff
@@ -87,12 +87,24 @@
...
+  bottom: 20px;
+  right: 0;
+  left: 0;
+  width: 34px;
+  height: 40px;
+  position: absolute;
+  margin: auto;
+}
+
+.homepage-hero-scroll__lookdown{
```
Comment (hanna446) [inline css/home.css]: Please add a space between	**.homepage-hero-scroll__lookdown** and **{**

File: css/home.css [hunk 2]
```diff
@@ -182,6 +191,14 @@
     max-width: 350px;
     height: 198px;
   }
+
+  .homepage-hero-scroll__lookdown{
```
Comment (camigomezdev) [inline css/home.css]: please change this name to `.homepage-hero__anchor`

File: js/index.js [hunk 3]
```diff
@@ -94,10 +94,8 @@ carrouselArrows.forEach(function (element) {
 /* Function about the arrow down or drop down wherever you want
 */
 function goDown() {
-  console.log('click');
-  
-  let lookdown = document.getElementById('carrousel-section');//take the destini to scroll
-  lookdown.scrollIntoView({ block: "start", behavior: "smooth" });
+  let lookdown = document.getElementById('lookdown-scroll');//take the destini to scroll
```
Comment (camigomezdev) [inline js/index.js]: Change this name variable to `anchorElement` and Id to `homepage-hero__anchor`

File: js/index.js [hunk 4]
```diff
@@ -132,3 +130,4 @@ document.getElementById('divBtn4').addEventListener('click', function () {
 });
 document.getElementById('hero-lookdown').addEventListener('click', goDown);
 
+
```
Comment (camigomezdev) [inline js/index.js]: please delete this line.

File: css/home.css (hunk 1 above)
Comment (Sonny97) [reply css/home.css]: done

File: css/home.css (hunk 2 above)
Comment (Sonny97) [reply css/home.css]: done

File: js/index.js (hunk 3 above)
Comment (Sonny97) [reply js/index.js]: done

File: js/index.js (hunk 4 above)
Comment (Sonny97) [reply js/index.js]: this line called the event to scroll down, it's necesary

Comment (josemariogutierrez) [reply js/index.js]: She refers to the empty space, line 133

Comment (Sonny97) [reply js/index.js]: done

File: css/home.css [hunk 5]
```diff
@@ -90,11 +90,20 @@
...
   right: 0;
   left: 0;
-  width: 33px;
+  width: 34px;
+  height: 40px;
   position: absolute;
   margin: auto;
 }
 
+.homepage-hero-anchor {
```
Comment (jusk02) [inline css/home.css]: this class should be something like `homepage-hero__anchor` to follow bem conventions

Comment (Sonny97) [reply css/home.css]: done

**Review by jusk02 — APPROVED** [review]

**Review by camigomezdev — APPROVED** [review]

**Pull request merged**